pkgs/development/python-modules/fastcrc/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchPypi,
+  rustPlatform,
+  nix-update-script,
+}:
+let
+  pname = "fastcrc";
+  version = "0.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Tip1NZ2XpZRpu8vXdRPoAdPzZmqRaWKVePiAEsDy1/4=";
+  };
+in
+python3.pkgs.buildPythonPackage {
+  inherit pname version src;
+  pyproject = true;
+  disabled = python3.pkgs.pythonOlder "3.7";
+
+  nativeBuildInputs = with rustPlatform; [
+    cargoSetupHook
+    maturinBuildHook
+  ];
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-HSYntwCIHtfAaIdEqw7kofO3lpeYcCiGlafOgI5K7mk=";
+  };
+
```

Comment:
Could you enable tests with `pytestCheckHook` and `pythonImportsCheck`?

Reply:
The tests don't seem to work (even though the import check does)... I'm not too familiar with Maturin so maybe something about the import paths are messed up?
```console
       > ____________________ ERROR collecting tests/test_fastcrc.py ____________________
       > ImportError while importing test module '/build/source/tests/test_fastcrc.py'.
       > Hint: make sure your test modules/packages have valid Python names.
       > Traceback:
       > /nix/store/wfbjq35kxs6x83c3ncpfxdyl5gbhdx4h-python3-3.12.6/lib/python3.12/importlib/__init__.py:90: in import_module
       >     return _bootstrap._gcd_import(name[level:], package, level)
       > tests/test_fastcrc.py:3: in <module>
       >     from fastcrc import crc8, crc16, crc32, crc64
       > fastcrc/__init__.py:1: in <module>
       >     from . import crc8, crc16, crc32, crc64
       > fastcrc/crc8.py:6: in <module>
       >     from .fastcrc import crc_8_autosar as _crc_8_autosar
       > E   ModuleNotFoundError: No module named 'fastcrc.fastcrc'
```

Reply:
Yes, an uncompiled Python file may block the import.
We usually remove the python files from the source (rm -r fastcrc) or run `cd $out` in preCheck phase.

Reply:
That does it! Many thanks ❤️